cmd/noms/noms_log.go

```diff
@@ -61,18 +64,27 @@ func setupLogFlags() *flag.FlagSet {
 func runLog(args []string) int {
 	useColor = shouldUseColor()
 	cfg := config.NewResolver()
-	database, value, err := cfg.GetPath(args[0])
 
-	if err != nil {
-		d.CheckErrorNoUsage(err)
+	resolved := cfg.ResolvePathSpec(args[0])
+	sp, err := spec.ForPath(resolved)
+	d.CheckErrorNoUsage(err)
+	pinned, ok := sp.Pin()
+	if !ok {
+		fmt.Fprintf(os.Stderr, "Cannot resolve spec: %s\n", args[0])
+		return 1
 	}
+	database := pinned.GetDatabase()
 	defer database.Close()
 
-	if value == nil {
-		d.CheckErrorNoUsage(fmt.Errorf("Object not found: %s", args[0]))
+	absPath := pinned.Path
+	d.Chk.NotEmpty(absPath.Hash) // because we pinned it
```

Comment:
better to be in the habit of doing `d.PanicIfTrue` even though in this case the interface boxing isn't a big deal. But this assertion doesn't seem so useful.